QUICKSTART.md

```diff
@@ -47,10 +47,12 @@ Tracer tracer =
 To create a basic span, you only need to specify the name of the span.
 The start and end time of the span is automatically set by the OpenTelemetry SDK.
 ```java
-Span span = tracer.spanBuilder("SpanName").startSpan();
-// your use case
-...
-span.end();
+Span span = tracer.spanBuilder("my span").startSpan();
+try (Scope scope = tracer.withSpan(span)) {
+	// your use case
+	...
+}
+span.end(); // closing the scope does not end the span, this has to be done manually
```

Comment:
I would put this in a `finally` block to ensure always called. Also we may have a `catch` block where we set a `status`

Reply:
I wanted to do it at start, but name of section (### Create basic Span) confused me